src/Http/Http/src/HttpContextFactory.cs

```diff
@@ -8,28 +8,51 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
+    /// Represents methods used to create an HTTP context object. 
+    /// <summary/>
     [Obsolete("This is obsolete and will be removed in a future version. Use DefaultHttpContextFactory instead.")]
```

Comment:
Not sure we need a lot of docs here if the type is obsolete. We might just delete the type entirely in 5.0

Reply:
I figured it would not hurt to have them.